src/WinRT.Runtime/ExceptionHelpers.Windows.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Windows.UI.Xaml
```

Comment:
Probably want NET guard for WUX version to avoid any netstandard conflicts.

Reply:
Fixed 👍